### ISSUE TITLE
Contradictory license and copying files

### ISSUE THREAD

**Issue by alichnewsky** [FrancescAlted/caterva#115]
`COPYING` file is GNU GPL
`LICENSE` file is BSD like. 
the documentation overall is [kind of hinting](https://caterva.readthedocs.io/en/latest/development/contributing.html?highlight=license#contributing-to-code) that the LICENSE file is the one for _further contributions_

I am not the only one confused as Github classifies the project as  `Unknown, GPL-2.0 licenses found`

can you please clarify ?

**Comment by FrancescAlted** [comment]
Yes, the intent was to allow projects to use the library under the license they prefer.  We are using the same approach than https://github.com/facebook/zstd.  See some discussion about this here: https://github.com/facebook/zstd/issues/1181

**Comment by alichnewsky** [comment]
Thanks for pointing that out so promptly. I indeed could have read the four files of implementation. 
I don't mind closing the issue, unless you intend to change the README or the documentation, and want it to stay opened.
